pennylane/_qubit_device.py

```diff
@@ -477,24 +477,25 @@ def marginal_prob(self, prob, wires=None):
             return prob
 
         wires = Wires(wires)
-        # get indices of wires on the device's register
-        wire_indices = self.register.indices(wires)
-
         # determine which subsystems are to be summed over
-        inactive_registers = list(set(range(self.num_wires)) - set(wire_indices))
+        inactive_wires = Wires.unique_wires([self.wires, wires])
```

Comment:
@co9olguy I think `\mathbb{P}^{(2, 0)}` is just bad/misuse of notation and should be removed/replaced.

Essentially, what this docstring is trying to say is that, if you request the probabilities of wires `[1, 2, 0]` (for example), it first computes the probabilities as normal (corresponding to `[0, 1, 2]`), and then performs a permutation of the probability array to match the users requested wire order.

This is required to allow cases where the user applies operations to non-consecutive wires. E.g., `expval(qml.Hermitian(A, wires=[0, 1]))` is a different result to `expval(qml.Hermitian(A, wires=[1, 0]))`, and this difference is taken into account by using the same matrix `A` in both cases, but permuting the probabilities.

Reply:
To answer your first question:

I think this functionality is still required, however it will be applied with respect to the ordering of the device's internal labelling scheme.

Reply:
I rewrote just the text to

            If the provided wires are not in the order as they appear on the device, 
            the returned marginal probabilities take this permutation into account.

            For example, if the addressable wires on this device are ``Wires([0, 1, 2])`` and 
            this function gets passed ``wires=[2, 0]``, then the returned marginal
            probability vector will take this 'reversal' of the two wires
            into account